.github/teams.yml

```diff
@@ -0,0 +1,34 @@
+Krypton:
```

Comment:
No need to add this file. We only have it in murmur because murmur is a shared repo :)